Directory.Build.props

```diff
@@ -36,6 +36,11 @@
     <IncludeSymbols>true</IncludeSymbols>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NuGetAuditMode>all</NuGetAuditMode>
+    <NuGetAuditLevel>moderate</NuGetAuditLevel>
+  </PropertyGroup>
+    
```

Comment:
Are they enabling these by default in newer .NET SDKs anyway?

https://github.com/NuGet/NuGet.Client/blob/5485ea697de98eee58746e0b0054cd478e33a1a5/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets#L71-L86

I remember seeing `NuGetAudit` setup by default in .NET 9 at one point.